src/Modals/ModalBody.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+import styled, { withTheme } from 'styled-components';
+import { BaseTheme } from 'Theme';
+
+interface ModalBodyProps {
+  /**
+   * the content of the Modal Footer
```

Comment:
Did you mean 'Modal Body'?

Reply:
Yep, that's exactly right. fixed in 9eb9ebf